### DISCUSSION TITLE
有关自定义主页间接广告的定义

### DISCUSSION THREAD

**Discussion by LinQingYuu** [Meloong-Git/PCL#5208] [提问] [answered]
如题，有些百科站点会带一定量广告，例如萌娘百科（用于维持站点服务开销）
如果主页包含这类站点的链接是否算间接广告？

**Comment by Joker2184** [comment]
我觉得吧
处于自己能力范围的事 :D

当然愿意给机子续费也是好的（Orz

**Reply by LinQingYuu** [reply]
我其实指的是能不能 link 过去

**Reply by Joker2184** [reply]
我认为是可以的 毕竟不是特意为了广告而制作（

还是等信标或龙猫来看看

**Comment by Joker2184** [comment, marked as answer]
![QQ截图20241202180048](https://github.com/user-attachments/assets/c28cec8e-9bce-4c90-92dc-0a3a26ea0e28)
不算可以搞

**Reply by LinQingYuu** [reply]
那我可没任何心里负担了
开写！
